openquake/commonlib/export/hazard.py

```diff
@@ -727,7 +727,7 @@ def export_gmf_txt(key, dest, sitecol, imts, ruptures, rlz,
     return {key: [dest]}
 
 
-def get_rup_idx(ebrup, etag):
+def get_eidx(ebrup, etag):
```

Comment:
Thanks Paolo, actually I did not realize that `get_eidx` was not used anymore. I removed it now.